console/my-account/how-to/configure-support-plans.mdx

```diff
@@ -13,8 +13,6 @@ categories:
   - console
 ---
 
-Although all roles can open support tickets, only an Owner can alter the Organization's [support plan](/console/my-account/concepts/#support-plan/). 
-
 <Macro id="iam-requirements" />
```

Comment:
We need to change the text of this macro to remove the "If you have activated IAM..." part

Reply:
DELETE THE START OF THE MACRO